tests/build/index.tst.ts

```diff
@@ -1,6 +1,6 @@
 import * as aws from '@pulumi/aws';
 import * as awsx from '@pulumi/awsx';
-import { describe, expect, test as it } from 'tstyche';
+import { describe, expect, it } from 'tstyche';
```

Comment:
`it` is exported just like `test`. You can use one or another.

Reply:
Thanks for the tip. I missed that part in the docs.